Utilities/import-llvm

```diff
@@ -189,11 +305,52 @@ def main():
     ]
 
     include_dir = os.path.join(sourcekit_srcroot, 'include')
-
     for base in base_dirs:
         for root, dirs, files in os.walk(base):
             for file in files:
                 maybe_add_prefix(os.path.join(root, file), include_dir)
 
+    # Finish off with a bunch of really gross hacks. These all offend my better
+    # sensibilities and what have you.
+
+    # TODO: Indexstoredb uses an out of tree/non merged OptionSet.h We should
+    # remove uses of this or get it merged upstream.
+    copyfile(os.path.join(sourcekit_srcroot, 'Utilities', 'import-llvm.d', 'include', 'llvm', 'ADT', 'OptionSet.h'),
+             os.path.join(sourcekit_srcroot, 'include', 'llvm', 'ADT', 'OptionSet.h'))
+
+    # AArch64TargetParser uses a relative path to find the .def file, so we
+    # copy this one to the expected location :(
+    def_location = os.path.join(sourcekit_srcroot, 'lib', 'include', 'llvm', 'Support')
+    if os.path.exists(def_location):
+        shutil.rmtree(def_location)
+    os.makedirs(def_location)
+    copyfile(os.path.join(sourcekit_srcroot, 'include', 'llvm', 'Support', 'AArch64TargetParser.def'),
+             os.path.join(def_location, 'AArch64TargetParser.def'))
+
+    # Timer.cpp and Statistics.cpp use YAMLTraits to assert some JSON is valid. We don't want to
+    # pull in the YAML/Regex so we'll remove the "offending" lines.
+    timer_location = os.path.join(sourcekit_srcroot, 'lib', 'LLVMSupport', 'Support', 'Timer.cpp')
```

Comment:
Yikes, this is pretty gross.  Could we use a patch file instead of doing this with `sed`?

Reply:
Sure, that seems like a much nice way to do it.